src/style.css

```diff
@@ -0,0 +1,116 @@
+* {
+  margin: none;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+.card {
+  display: flex;
+  justify-content: space-around;
+}
+
+.pic {
+  height: 134px;
+  width: 160px;
+  margin: 32px 19px;
+}
+
+.card__cards {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__title {
+  font-weight: 500;
+  margin: 8px 16px;
+  text-align: center;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__code {
+  margin: 2px 16px 0;
+  text-align: left;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.star__text {
+  display: inline-flex;
```

Comment:
why have u used this property?

Reply:
I used it, because I wanted  text "Reviews: 5" and stars be in the same line